config/dynamic.c

```diff
@@ -464,11 +462,21 @@ set_dyc_timeout(unsigned int sec,
 }
 
 int
-run_dynamic_config(const char *server_path, struct dynamic_config *dy_conf)
+run_dynamic_config(struct gk_config *gk_conf,
+	const char *server_path, struct dynamic_config *dy_conf)
 {
 	int ret;
 	struct sockaddr_un server_addr;
 
+	/*
+	 * When the dynamic configuration is run for Gatekeeper,
+	 * the gt_conf should be NULL.
+	 * When the dynamic configuration is run for Grantor,
+	 * the gk_conf should be NULL.
+	 * The code works fine for both situations.
```

Comment:
Replacing `The code works fine for both situations.` with `The code works fine with both being NULL as well.` emphasizes the generality of the code.